### ISSUE TITLE
Activation of output layer

### ISSUE THREAD

**Issue by sobhanhemati** [vanHavel/bp-mll-tensorflow#4]
Hi,

Just noticed in the paper activation of the output layer is tanh while in your example you used sigmoid.

Is it possible to use both?

**Comment by vanHavel** [comment]
Sorry for the late reply.
To be honest, I did not give this much thought when writing the example. 

In principle it is of course possible to use both. Personally, I would treat the output activation as a hyperparameter that needs to be tuned. It might be that one or the other works better in general, it might be that it varies by dataset / task.

The range of the functions is different ([-1, 1] for tanh, [0, 1] for sigmoid) so you will also need to adapt the thresholding strategy when doing classification.

**Comment by vanHavel** [comment]
If there are no more questions, I will close this issue. Feel free to reopen it otherwise!
